docs/man_pages/lib-management/kendoui-notes.md

```diff
@@ -0,0 +1,49 @@
+kendoui notes
+==========
+
+Usage | Synopsis
+------|-------
+All versions | `$ appbuilder kendoui notes [--verified] [--latest]`
+Only Kendo UI Core versions | `$ appbuilder kendoui notes --core [--verified]`
+Only Kendo UI Professional versions | `$ appbuilder kendoui notes --professional [--verified]`
+
+Shows release notes for selected Kendo UI Core or Kendo UI Professional package.<% if(isHtml || isCordova) { %> The verified tag marks stable Kendo UI Service Pack releases.<% } %> 
```

Comment:
When running this command the packages are filtered based on whether or not they have release notes. So I'm guessing 'If available' should be omitted?